docs_src/microservices/V2TopLevelMigration.md

```diff
@@ -0,0 +1,97 @@
+# V2 Migration Guide
+
+!!! edgey "EdgeX 2.0"
+    Many backward breaking changes occurred in the EdgeX 2.0 (Ireland) release which may require some migration depending on your use case.
+
+This section describes how to migrate from EdgeX 1.x to EdgeX 2.0 at a high level and refers the reader to the appropriate detail documents. The areas to consider for migrating are:
+
+- [Compose File](#compose-file)
+- [Database](#database)
+- [Custom Configuration](#custom-configuration)
+- [Custom Device Service](#custom-device-service)
+- [Custom Applications Service](#custom-applications-service)
+
+## Custom Compose File
+
+The compose files for V2 have many changes from their V1 counter parts. If you have customized a V1 compose file to add additional services or tweak overrides,  it is highly recommended that you start with the appropriate V2 compose file and re-add your customizations. It is very likely that the sections for your additional services will need to be migrated to have the proper environment overrides. Best approach is to use one of the V2 service sections that closest matches your service (App or Device) as a template.
+
+## Database
+
+There currently is no migration path for the data stored in the database. The V2 data collections are stored separately from the V1 data collections in the Redis database. Redis is now the only supported database, i.e. support for Mongo has been removed.
```

Comment:
Have we agreed to this?  I thought we were going to try to document all the effort and then chat about whether a tool or migration document was required.  Sensor data (core data) is unlikely to be migrated, but I could see where someone might want to migrate metadata.

Reply:
That is why is used `currently`. Figured that if we created a tool we would change this and reference the tool. Are you thinking we do a detailed migration document outlining how to migrate the DB?